sources/engine/Stride.Games/GameTime.cs

```diff
@@ -32,6 +32,7 @@ public class GameTime
     {
         private TimeSpan accumulatedElapsedTime;
         private int accumulatedFrameCountPerSecond;
+        private static double factor;
```

Comment:
Users can use stride to run multiple game instances concurrently in the same process, factor should not be static otherwise all game instances will share the same factor.

Reply:
I've tried this first, one of my issue with it is that Particle and Animation systems use a RenderContext GameTime which is not a reference to the GameTime accessible in scripts.
My workaround was doing that in a sync script : 
```csharp
if(Input.IsKeyPressed(Keys.U))
{
        slowdown = !slowdown;
        Game.UpdateTime.Factor = slowdown ? 0.5f : 1;
        var rc = RenderContext.GetShared(SceneSystem.Services);
        rc.Time.Factor = slowdown ? 0.5f : 1;
}
```

Reply:
Alright, after taking a closer look you shouldn't have to change stuff in the animation and particle system; you change the update time's factor but not the draw time and that variable is passed onto the rendercontext/animation and particle system so if both the draw and update time factors are the same you shouldn't have to bother with including the factor mult in the animation and particle file.

But yeah, users shouldn't have to modify both factor for it to work correctly though, it doesn't make a lot of sense to have the game's logic slowdown but not the animation and particle so I'll create a PR after yours to merge update and draw time together, I don't see any reason to keep them separate.

So all you should do is replace Elapsed by WarpElapsed for animation, particle and physics (https://github.com/stride3d/stride/blob/aab1aa8fca05210fbfee0a0b0c33b17abb300419/sources/engine/Stride.Physics/Bullet2PhysicsSystem.cs#L95), replace static fields with instance fields, match the drawtime's factor with the update time's factor in gamebase (around here https://github.com/stride3d/stride/blob/aab1aa8fca05210fbfee0a0b0c33b17abb300419/sources/engine/Stride.Games/GameBase.cs#L635), and if that works you're good to go.

Reply:
Thanks for your insight and your time!